driver/src/util.rs

```diff
@@ -111,14 +111,17 @@ pub fn batch_processing_state(
     Ok(ProcessingState::TooEarly)
 }
 
-pub fn create_price_finder(fee: Option<Fee>) -> Box<dyn PriceFinding> {
-    let solver_env_var = env::var("LINEAR_OPTIMIZATION_SOLVER").unwrap_or_else(|_| "0".to_string());
+pub fn create_price_finder<N>(fee: Option<Fee>, naive: N) -> Box<dyn PriceFinding>
```

Comment:
Maybe this method should now just return an Option< LinearOptimisationPriceFinder > and then we unwrap_or_else it with the naive solver?

Reply:
Yeah that would have been much better.

Reply:
I will address this when I merge the naive solvers.